src/services/serverFunctions/auth.ts

```diff
@@ -22,6 +22,11 @@ export const checkToken = async (token: string) => {
   }
 }
 
+export const checkPassword = async (email: string) => {
```

Comment:
```suggestion
export const hasEmptyPassword = async (email: string) => {
```

Une fonction checkPassword me fait penser à une vérification mdp entré / mdp enregistré

Reply:
C'est ok